libs/java/server_common/src/main/java/com/yahoo/athenz/common/server/store/AWSCredentialsRefresher.java

```diff
@@ -0,0 +1,111 @@
+/*
+ *  Copyright 2020 Verizon Media
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package com.yahoo.athenz.common.server.store;
+
+import com.amazonaws.auth.AWSCredentials;
+import com.yahoo.athenz.common.server.util.ConfigProperties;
+import com.yahoo.athenz.zms.ResourceException;
+import org.eclipse.jetty.util.StringUtil;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.Closeable;
+import java.util.concurrent.Executors;
+import java.util.concurrent.ScheduledExecutorService;
+import java.util.concurrent.TimeUnit;
+
+import static com.yahoo.athenz.common.ServerCommonConsts.ZTS_PROP_AWS_CREDS_UPDATE_TIMEOUT;
+import static com.yahoo.athenz.common.ServerCommonConsts.ZTS_PROP_AWS_REGION_NAME;
+
+public class AWSCredentialsRefresher implements Closeable {
```

Comment:
Extracted this class from CloudStore. It gets credentials and region from AWSInstanceMetadataFetcher (also a new class) and has a periodic task that refreshes the credentials.
Note that in the CloudStore, this task was responsible for 2 things:
1. Refresh the credentials
2. Clear the CloudStore Credentials Cache.

As they are now separated, there are 2 unrelated periodic tasks - one here to refresh the credentials and the other remained in the CloudStore